cluster/CHAPTER.rst

```diff
@@ -846,18 +835,16 @@ It responds to the leader with either ``Decided`` or ``Preempted``.
     
 }}}
 
-.. note::
 
-    A surprisingly subtle bug appeared here during development.
```

Comment:
We're not including sidebars in this book, so I removed the indentations here and added some text ("As an aside") to allow it to flow as a regular paragraph.